### ISSUE TITLE
Birth cooldown time algorithm problem?

### ISSUE THREAD

**Issue by DopiePanda** [jasonrohrer/OneLife#1131]
So if we look at the code for birth cooldown;
https://github.com/jasonrohrer/OneLife/blob/master/server/server.cpp#L2091

We can see the the equation is set as the following

a = 2
b = 3
u = randSource.getRandomDouble(); (This returns a double between 0 and 1.0)
v = pow( 1 - pow( 1-u, (1/b) ), 1/a );
x = v * 5 * 60

If 'u' in the equation above equals 0, the "pickBirthCooldownSeconds" function will output 0, as seen here:
![image](https://github.com/user-attachments/assets/fd469478-0f52-4c05-b683-641efd356d87)

Increasing 'u' to 0.1 increases the cooldown timer to 55.731099 seconds, as seen here;
![image](https://github.com/user-attachments/assets/acc84cfd-5754-419e-acff-419328389237)

And if the 'getRandomDouble' function returns it's maximum value (1), it will increase the cooldown timer to 300 seconds, as seen here;
![image](https://github.com/user-attachments/assets/a1cd35cb-c88f-4628-a38b-89446e910dd1)

**Comment by DopiePanda** [comment]
I'm not sure if this is intended or not, as I was under the impression that there was a "minimum birth cooldown".

**Comment by oholseven** [comment]
This change could solve all of Strilar's rants on birth cooldown.

**Comment by jasonrohrer** [comment]
No, it's intentionally supposed to have the full range of 0-1, which is then applied to a time of up to 5 minutes.  But sometimes, very rarely, there are "Irish Twins" born very close together, just for variety.
